package.json

```diff
@@ -69,9 +71,10 @@
     "fixcjs": "fs.writeFileSync('cjs/writeFile.js', fs.readFileSync('cjs/writeFile.js').toString().replace(`(await Promise.resolve().then(function () {\n  return _interopRequireWildcard(require('node:fs'));\n})).default`, `require('node:fs')`));fs.writeFileSync('cjs/readFile.js', fs.readFileSync('cjs/readFile.js').toString().replace(`(await Promise.resolve().then(function () {\n  return _interopRequireWildcard(require('node:fs'));\n})).default`, `require('node:fs')`))"
   },
   "scripts": {
+    "copy:jsonc": "esbuild node_modules/jsonc-parser/lib/esm/main.js --bundle --format=esm --platform=neutral --banner:js=\"/*\n$(cat node_modules/jsonc-parser/LICENSE.md)*/\" --outfile=lib/formats/jsonc.js",
```

Comment:
Unlike `json5` and `js-yaml`, `json-parser` does not ship with a `dist` directory we can copy a single file out of.

To work around this, we use ESBuild to produce a single file we can vendor, ensuring to include a copy of the license, as required by its terms.